libweb3jsonrpc/Debug.cpp

```diff
@@ -95,6 +96,19 @@ Json::Value Debug::debug_traceTransaction(string const& _txHash, Json::Value con
 	return ret;
 }
 
+Json::Value Debug::debug_traceBlock(string const& _blockRLP, Json::Value const& _json)
+{
+	bytes bytes = fromHex(_blockRLP);
+	BlockHeader blockHeader(bytes);
+	return debug_traceBlockByHash(blockHeader.hash().hex(), _json);
+}
+
+Json::Value Debug::debug_traceBlockFromfile(string const& _fileName, Json::Value const& _json)
+{
+	string content = contentsString(_fileName);
```

Comment:
Hm, I would say let's not expose the complete filesystem via the debug interface, people don't expect that.